code/include/playfab/PlayFabClientAuthApi.h

```diff
@@ -0,0 +1,352 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if !defined(__cplusplus)
+#error C++11 required
+#endif
+
+#pragma once
+
+#include <playfab/PlayFabAuth.h>
+#include <playfab/PlayFabClientDataModels_c.h>
+
+extern "C"
+{
+
+/// <summary>
+/// <LoginWithAndroidDeviceID documentation goes here.>
+/// </summary>
+/// <param name="stateHandle">PlayFabStateHandle returned from PlayFabInitialize call.</param>
+/// <param name="request">Populated request object.</param>
+/// <param name="async">XAsyncBlock for the async operation.</param>
+/// <returns>Result code for this API operation.</returns>
+/// <remarks>
+/// If successful, call <see cref="PlayFabClientGetAuthResult"/> to get the result.
+/// </remarks>
+HRESULT PlayFabClientLoginWithAndroidDeviceID(
+    _In_ PlayFabStateHandle stateHandle,
+    _In_ const PlayFabClientLoginWithAndroidDeviceIDRequest* request,
```

Comment:
>PlayFabClientLoginWithAndroidDeviceIDRequest [](start = 15, length = 44)

Where are these defined?

Reply:
PlayFabClientDataModels_c.h

---
In reply to: [607290987](https://github.com/jplafonta/XPlatCSdk/pull/2#discussion_r607290987) [](ancestors = 607290987)